packages/suite/src/config/suite/yup.ts

```diff
@@ -0,0 +1,22 @@
+import { isHex as isHexUtil } from '@suite/utils/wallet/ethUtils';
+import { isAscii as isAsciiUtil } from '@trezor/utils';
+import * as yup from 'yup';
+
+yup.setLocale({
+    string: {
+        max: 'TR_EXCEEDS_MAX',
+    },
+    mixed: {
+        required: 'TR_REQUIRED_FIELD',
+    },
+});
+
+yup.addMethod<yup.StringSchema>(yup.string, 'isAscii', function isAscii() {
```

Comment:
why are you not using arrow function? it will be better because you will not need to rename imported validation functions

Reply:
It's an old, unnecessary pattern, using `this`. Changed to arrow funcs.